### PR TITLE
Pvc

### PR DESCRIPTION
**Pull request by AjeshStavir**
Added some CSS styling.

### CODE REVIEW

File: Present Value Calculator/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+function presentvalue(fv,interest,years){
```
Comment (skStavir) [inline Present Value Calculator/index.js:1]: why do we need this?

File: tools/emi-calculator/emi-calculator.html [hunk 2]
```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
```
Comment (skStavir) [inline tools/emi-calculator/emi-calculator.html:1]: Why is this code updated now?
Was it not already checked in?

**Comment by skStavir** [comment]
Next time use a more meaningful commit message that has a brief description of the change

File: Present Value Calculator/index.js (hunk 1 above)
Comment (AjeshStavir) [reply Present Value Calculator/index.js:1]: this code wasn't meant to include in the original html file . It was only meant for testing my code ,So we don,t need it !

File: tools/emi-calculator/emi-calculator.html (hunk 2 above)
Comment (AjeshStavir) [reply tools/emi-calculator/emi-calculator.html:1]: As I was learning the pr process all the files got added to the branch I created so next time the specific files only be committed.

File: Present Value Calculator/index.js (hunk 1 above)
Comment (skStavir) [reply Present Value Calculator/index.js:1]: delete it then

**Review by skStavir — APPROVED** [review]

**Comment by skStavir** [comment]
why is this still not merged?

**Pull request merged**